frontend/src/utils/server-utils.ts

```diff
@@ -420,6 +420,25 @@ const layerDefinitionsBluePrint: AvailableDates = Object.keys(
   };
 }, {});
 
+/**
+ * Load preprocessed date ranges if available
+ * */
+async function fetchPreprocessedDates(): Promise<any> {
+  try {
+    const response = await fetch(
+      `data/${safeCountry}/preprocessed-layers.json`,
```

Comment:
I always find it handy when there's a comment explaining where these files are generated :)